projects/ngrx.io/content/guide/data/entity-reducer.md

```diff
@@ -0,0 +1,241 @@
+# Entity Reducer
+
+The _Entity Reducer_ is the _master reducer_ for all entity collections in the stored entity cache.
+
+<a id="reducer-factory"></a>
+
+The library doesn't have a named _entity reducer_ type.
+Rather it relies on the **`EntityReducerFactory.create()`** method to produce that reducer,
+which is an _NgRx_ `ActionReducer<EntityCache, EntityAction>`.
+
+Such a reducer function takes an `EntityCache` state and an `EntityAction` action
+and returns an `EntityCache` state.
+
+The reducer responds either to an [EntityCache-level action](#entity-cache-actions) (rare)
+or to an `EntityAction` targeting an entity collection (the usual case).
+All other kinds of `Action` are ignored and the reducer simply returns the given `state`.
+
+> The reducer filters specifically for the action's `entityType` property.
```

Comment:
It seems like we haven't styled quote blocks, in other parts of the docs we're creating alerts, for example:


```
<div class="alert is-important">

An **entity** is an object with long-lived data values that you read from and write to a database.  An entity refers to some "thing" in the application domain.  Examples include a _Customer_, _Order_, _LineItem_, _Product_, and _User_.

</div>
```

We have the following types available:
- `is-critical`
- `is-important`
- `is-helpful`

Reply:
Thanks for pointing that out. I swept these docs for quote blocks and hope I've got them all.